supervision/dataset/formats/pascal_voc.py

```diff
@@ -19,13 +19,13 @@ def object_to_pascal_voc(
 
     bndbox = SubElement(root, "bndbox")
     xmin = SubElement(bndbox, "xmin")
-    xmin.text = str(int(xyxy[0]))
+    xmin.text = str(int(xyxy[0] + 1))
```

Comment:
Wouldn't we get away with `xyxy += 1` at the top of the function body?

Reply:
Wow, not sure how I missed that :p. Thanks for pointing out.